src/MessageBus.ts

```diff
@@ -30,8 +32,9 @@ export default class MessageBus<Msg> {
     this.sendQ.push(msg)
   }
 
-  subscribe(onMsg: (msg: Msg) => void): void {
+  subscribe(onMsg: (msg: Msg) => void): this {
     this.receiveQ.subscribe(onMsg)
+    return this
```

Comment:
Why return `this`?

Reply:
I wanted to use `const bus = new MessageBus(...).subscribe(...)`, but then I added `subscriber?` to the constructor, so I guess it's not really needed anymore.